Seminars/Sem.07/Pract. 07/MyString.cpp

```diff
@@ -0,0 +1,479 @@
+#include "MyString.h"
+#include "Utils.h"
+
+// Big 4
+MyString::MyString() : MyString(nullptr) {}
+
+MyString::MyString(const char *data) : _data(new char[DEFAULT_CAPACITY]{'\0'}), _capacity(DEFAULT_CAPACITY)
```

Comment:
Why do you allocate memory twice, this could be done just once in the body of the constructor